.github/workflows/build.yml

```diff
@@ -143,6 +143,9 @@ jobs:
             find ./site-docs/generated -type f -exec grep -L "." {} \; >&2
             exit 1
           fi
+      - name: Check missing lib license
+        run: |
+          ./gradlew releaseTarGz
```

Comment:
I think @mumrah means, this gradle command should move to `Compile and validate` step